getting-started-sdks.md

```diff
@@ -46,12 +46,34 @@ The following {{site.data.keyword.watson}} SDKs are supported by {{site.data.key
 * [Swift SDK](https://github.com/watson-developer-cloud/swift-sdk){: external}
 * [Unity SDK](https://github.com/watson-developer-cloud/unity-sdk){: external}
 
-The [API reference](https://{DomainName}/apidocs?category=ai){: external} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, Go, and Ruby.
+The [API reference](https://{DomainName}/apidocs?category=ai){: external} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, Go , .Net, Ruby, and Swift.
 
-## Community SDKs
+<!-- ## Community SDKs
 {: #community-sdks}
 
 The following SDKs are available from the {{site.data.keyword.watson}} community of developers:
+ -->
 
-* [PHP SDK](https://github.com/CognitiveBuild/WatsonPHPSDK){: external}
-* [Scala SDK](https://github.com/kane77/scala-sdk){: external}
+## SDK updates and deprecation
+{: #sdk-schedule}
+
+The supported {{site.data.keyword.watson}} SDKs are updated according to the following guidelines.
+
+### Semantic versioning
+
+Supported {{site.data.keyword.watson}} SDKs adhere to semantic versioning with releases labeled as `major.minor.patch`.
```

Comment:
Just a question:  Would it make sense to show these as `{major}.{minor}.{patch}`, since they're variable values?